CMakeLists.txt

```diff
@@ -604,7 +604,9 @@ if(WITH_SYSTEMD)
   add_subdirectory(systemd)
 endif()
 
-add_subdirectory(etc/sysctl)
+if(LINUX)
+  add_subdirectory(etc/sysctl)
```

Comment:
should'nt be /etc/sysctl

Reply:
no, it's relative to the top of the source directory.